source/includes/_ais.md

```diff
@@ -174,3 +174,14 @@ Note that the fields we return as part of the response depend on whether your co
 or `ReadAccountsDetailed` permission. In the former case, we will omit the Pot name and Image URL from the response.
 
 We'll only return open pots as part of our response.
+
+## Testing in the Sandbox
+
+In the **sandbox** environment, you can automatically have account information consents approved or declined to help with testing. To do this, you should set some specific fields in the `Data/Initiation/SupplementaryData` object:
+
+<code>
```

Comment:
Can we use a Markdown block so this appears on the right hand side of the screen?